### ISSUE TITLE
Release 4.1.1

### ISSUE THREAD

**Issue by garethbowen** [medic/cht-core#8116]
# Planning - Product Manager

- [x] Create an GH Milestone and add this issue to it.
- [x] Add all the issues to be worked on to the Milestone.
- [x] Ensure that all issues are labelled correctly, particularly ensure that "Regressions" are labelled with "Affects: <version>" labels. The "Affects" label is used in a link in the Known Issues section of the release notes of that version so it has to match exactly. To make sure the label is correct go to the [release notes](https://docs.communityhealthtoolkit.org/core/releases/#release-notes) and ensure the issue is listed.

# Development - Release Engineer

When development is ready to begin one of the engineers should be nominated as a Release Engineer. They will be responsible for making sure the following tasks are completed though not necessarily completing them.

- [x] Set the version number in `package.json` and `package-lock.json` and submit a PR to the release branch. The easiest way to do this is to use `npm --no-git-tag-version version patch`.
- [x] Write an update in the weekly Product Team call agenda summarising development and acceptance testing progress and identifying any blockers (the [milestone-status](https://github.com/medic/support-scripts/tree/master/milestone-status) script can be used to get a breakdown of the issues). The Release Engineer is to update this every week until the version is released.

# Releasing - Release Engineer

Once all issues have passed acceptance testing and have been merged into `master` and backported to the release branch release testing can begin.

- [x] Build a beta named `<major>.<minor>.<patch>-beta.1` by pushing a git tag and when CI completes successfully notify the QA team that it's ready for release testing.
- [x] Add release notes to the [Core Framework Releases](https://docs.communityhealthtoolkit.org/core/releases/) page:
    - [x] Create a new document for the release in the [releases folder](https://github.com/medic/cht-docs/tree/main/content/en/core/releases).
    - [x] Ensure all issues are in the GH Milestone, that they're correctly labelled (in particular: they have the right Type, "UI/UX" if they change the UI, and "Breaking change" if appropriate), and have human readable descriptions.
    - [x] Use [this script](https://github.com/medic/cht-core/blob/master/scripts/release-notes) to export the issues into our release note format.
    - [ ] Manually document any known migration steps and known issues.
    - [x] Add a link to the new release page in the [Release Notes](https://docs.communityhealthtoolkit.org/core/releases/#release-notes) section.
    - [x] Assign the PR to:
        - The Director of Technology
        - An SRE to review and confirm the documentation on upgrade instructions and breaking changes is sufficient
- [x] Until release testing passes, make sure regressions are fixed in `master`, cherry-pick them into the release branch, and release another beta.
- [x] Create a release in GitHub from the release branch so it shows up under the [Releases tab](https://github.com/medic/cht-core/releases) with the naming convention `<major>.<minor>.<patch>`. This will create the git tag automatically. Link to the release notes in the description of the release.
- [x] Confirm the release build completes successfully and the new release is available on the [market](https://staging.dev.medicmobile.org/builds_4/releases). Make sure that the document has new entry with `id: medic:medic:<major>.<minor>.<patch>`
- [x] Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org/), under the "Product - Releases" category using this template:
```
*Announcing the release of {{version}}*

This release fixes {{number of bugs}}. Read the [release notes]({{url}}) for full details.
```
- [ ] Mark this issue "done" and close the Milestone.

**Comment by garethbowen** [comment]
This release is to add Luganda translations to 4.1.x ( #7712 ). This is unconventional because that's not a bug fix and so should not be backported. However it's a low risk change and we're breaking the rules here as a one-off to help out our Uganda team.

**Comment by andrablaj** [comment]
@garethbowen the [scalability workflow](https://github.com/medic/cht-core/actions/workflows/scalability.yml) failed on the beta tag, and I see that it is a common thing in the last tags. You might already be aware of an issue there, but I prefer to notify you here anyways.

**Comment by garethbowen** [comment]
Thanks. Yes, I think that's caused by [this issue](https://github.com/medic/cht-core/issues/8024).

**Comment by andrablaj** [comment]
@garethbowen, the [build for 4.1.1](https://github.com/medic/cht-core/actions/runs/4435509750) is failing, and it seems like the failing tests are always the same for `integration` and `default`. 

I managed to reproduce the `/home/runner/work/cht-core/cht-core/tests/integration/api/routing.spec.js:808:53` failing test when running the tests locally on the release branch. Locally I got extra failing tests that passed in the build. 

I need more than my current understanding of the codebase to figure out why that test is failing. I can dig into it, but it might take a while. Can you please support this?

**Comment by garethbowen** [comment]
@andrablaj I'm sorry but I'm overloaded and won't be able to get to this today. I'm 99% sure those failures are bogus...

**Comment by garethbowen** [comment]
@andrablaj Yes feel free to cherry pick those fixes and any others you need into the `4.1.x` branch. Once that's passing you'll need to delete the release and the git tag and then recreate the release and tag on the branch.

**Comment by garethbowen** [comment]
@andrablaj Yes I think this can be safely ignored. Thanks!
